src/zx_plot.jl

```diff
@@ -0,0 +1,208 @@
+using ZXCalculus
+using LightGraphs
+using GraphPlot: gplot
+using Colors
+using ZXCalculus: qubit_loc
+using Compose
+
+export ZXplot
+
+function Multigraph2Graph(mg::Multigraph)
+    g = SimpleGraph(nv(mg))
+    vs = vertices(mg)
+    for me in edges(mg)
+        add_edge!(g, searchsortedfirst(vs, src(me)), searchsortedfirst(vs, dst(me)))
+    end
+    # multiplicities = ["$(mul(mg, src(e), dst(e)))" for e in edges(g)]
+    multiplicities = ["×$(mul(mg, vs[src(e)], vs[dst(e)]))" for e in edges(g)]
+    for i = 1:length(multiplicities)
+        if multiplicities[i] == "×1"
+            multiplicities[i] = ""
+        end
+    end
+    return g, multiplicities
+end
+
+ZX2Graph(zxd::ZXDiagram) = Multigraph2Graph(zxd.mg)
+ZX2Graph(zxg::ZXGraph) = Multigraph2Graph(zxg.mg)
+
+function et2color(et::String)
+    et == "" && return colorant"black"
+    et == "×2" && return colorant"blue"
+end
+
+function st2color(S::SpiderType.SType)
+    S == SpiderType.Z && return colorant"green"
+    S == SpiderType.X && return colorant"red"
+    S == SpiderType.H && return colorant"yellow"
+    S == SpiderType.In && return colorant"lightblue"
+    S == SpiderType.Out && return colorant"gray"
+end
+
+ZX2nodefillc(zxd) = [st2color(zxd.st[v]) for v in vertices(zxd.mg)]
+
+function ZX2nodelabel(zxd)
+    nodelabel = String[]
+    for v in vertices(zxd.mg)
+        zxd.st[v] == SpiderType.Z && push!(nodelabel, "[$(v)]\n$(print_phase(zxd.ps[v]))")
+        zxd.st[v] == SpiderType.X && push!(nodelabel, "[$(v)]\n$(print_phase(zxd.ps[v]))")
+        zxd.st[v] == SpiderType.H && push!(nodelabel, "[$(v)]")
+        zxd.st[v] == SpiderType.In && push!(nodelabel, "[$(v)]")
+        zxd.st[v] == SpiderType.Out && push!(nodelabel, "[$(v)]")
+    end
+    return nodelabel
+end
+
+function print_phase(p)
+    if typeof(p) <: Rational
+        return "$(p.num)π/$(p.den)"
```

Comment:
for the sake of consistency.
```suggestion
        return "$(p.num)π//$(p.den)"
```